packages/flutter/test/widgets/page_view_test.dart

```diff
@@ -1269,19 +1273,35 @@ void main() {
     controller.animateToPage(2, duration: const Duration(milliseconds: 300), curve: Curves.ease);
     await tester.pumpAndSettle();
 
-    final Finder transformFinder = find.descendant(
-      of: find.byType(PageView),
-      matching: find.byType(Transform),
-    );
-    expect(transformFinder, findsOneWidget);
+    if (isFragmentShader) {
+      final Finder effectFinder = find.descendant(
+        of: find.byType(PageView),
+        matching: find.byType(StretchOverscrollEffect),
```

Comment:
I think `StretchOverscrollEffect` should be considered implementation details (especially with [my comment](https://github.com/flutter/flutter/pull/169293/files#r2165791595) of making it private), and therefore should not be used to verify the result. In the end, what we care about is what the shader actually draws. Shall we instead write a few golden tests?